atlp-rwanda/commanders-medica-backoffice
# Feature: landing page design

#### What does this PR do?

This PR includes designs of the landing page.

#### Description of Task to be completed?

- Design landing page

#### How should this be manually tested?

Start the app with `npm run dev` and open the link. The landing page should be displayed

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A
